stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/querychannels/DefaultChatEventHandler.kt

```diff
@@ -47,12 +48,16 @@ internal class DefaultChatEventHandler(private val client: ChatClient, private v
     override fun handleNotificationAddedToChannelEvent(
         event: NotificationAddedToChannelEvent,
         filter: FilterObject,
-    ): EventHandlingResult = fireRequestIfChannelIsAbsent(event.channel, filter)
+    ): EventHandlingResult {
+        return handleMemberUpdate(event, event.cid, filter)
```

Comment:
In the previous version it did the same. Right?

Reply:
In this case it can remove the user in an Add events, accordingly with the filter, so the behaviour is a bit different now